mitodl/bootcamp-ecommerce
# render cms site wide notifications in react

#### What are the relevant tickets?
https://trello.com/c/Du27kx7o/176-render-cms-site-wide-notifications-in-react

#### What's this PR do?
fixes #544 , Render cms site wide notifications in react

#### How should this be manually tested?
Just add notification from cms and then will be able to see the notifications on the site

#### Screenshots (if appropriate)
<img width="1439" alt="Screenshot 2020-06-05 at 00 17 43" src="https://user-images.githubusercontent.com/4043989/83801156-17b7a900-a6c2-11ea-8b73-39cec5b7bccf.png">
<img width="1440" alt="Screenshot 2020-06-03 at 17 47 26" src="https://user-images.githubusercontent.com/4043989/83642562-71d44380-a5c8-11ea-90a2-5b273954fb0c.png">
<img width="1440" alt="Screenshot 2020-06-03 at 18 32 26" src="https://user-images.githubusercontent.com/4043989/83642686-97f9e380-a5c8-11ea-9eb5-ac9931511166.png">
